### DISCUSSION TITLE
MEETING: ecatalogue Working Group - 27 June 2022

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#604] [MEETINGS]
# Agenda: Relationships
- [Relationships Data & Workflows googledoc](https://docs.google.com/document/d/190msXiIYgui4zReB4O8-IQXq4Gue0GqQZU3wMCZG_jo/edit?usp=sharing) Relationships Tab
## Others

**Comment by fmjjones** [comment]
# Meeting Notes
**attendees: CM, WG, ED, JM, RSB,  KGJ, JK, LH, LJ, RD, SM, ZK, JJ**

## Use Cases for Relationship tabs vs Associations Tab

- In general, things to be published that relate in some way to "other" (External to the catalog record being edited) specimens/lots/objects belong on the Relationship Tab. Any management relationship and or internal to the specimens/lots/objects belong on the Associations Tab (parts, tissues, models of specimens, split lots, etc.) 
## Use Cases for Relationship tab vs Alternate Institution field workflow

- RRC uses the [workflow for non-FMNH records](https://docs.google.com/document/d/1GUGY4qwQEbon0dbVee1vst-Chw_hxL3GAopowiMQsys/edit?usp=sharing) to record images and data of other institutions specimens for use on the website.
- Exhibitions uses alternate institutions when creating catalog records for objects on loan, but still need to create an Exhibit Object for them.
- Mammals IRN 197620 as an example.

- Does anyone else use this workflow? If so, for what?
     - We will discuss this at the July 12th EMu users group meeting. We want to be able to publish other datasets with their stewarts permission without creating a duplicate occurrence record out in the world. 
## Workflow for updating existing relationships once new catalogue records have been created in another collection (e.g. Insects records updated with newly created Mammals records)
Mammals is creating a group and sharing with Insects that matches on Collectors number (which insects can currently see). Mammals sends a spreadsheet of specimens to Insects.
- Is there a way to “automate” both relationships when Insects have catalog records entered?
     - Beka will let me know when the parasite records (at least some are entered) Work out export/import for mammals side.
##  How to record on assemblage slabs? [PM] Github #591
- Paul wasn't here, but the same may be true for concretions in IZ. The question that needs to be answered is 2 fold:
    - Do we want to include them all as related and "publishable" on the relationship Tab (IZ is a yes)?
    - How to record the location of all but the specimen that it is filed under?

**Comment by zkachian** [comment]
Action put together [this doc](https://docs.google.com/document/d/1kfNReWjLa5kEv8zFPQJLPDJoflLTi9V0Ii_sZD7v-d4/edit?usp=sharing) with a list of questions we think are related to the Relationship and Associations tabs.

**Comment by wyattgaswick** [comment]
We use ObjRepository on the Object Tab in a relevant (?) way. Most records are just "F" because the specimens live here, but there are 3 main categories of other stuff:

1) Our record is a negative, in which case Repository indicates the institution where the object lived at the time of photography;
2) It's a rapid inventory record with the specimen somewhere else (I think all AMAZ)
3) Older bryophyte records for specimens that we received on loan. This is less relevant these days since most institutions have better databases than they used to. These could be deleted if needed

PS 4) We also have 4 records for INB (temporarily [?] defunct, we are holding specimens for them). We deleted most of these, but these 4 are attached to loan histories.
